XiEditor/Client.swift

```diff
@@ -54,6 +54,11 @@ protocol XiClient: AnyObject {
     /// A notification containing an alert message to be shown the user.
     func alert(text: String);
 
+    // Status bar notifications
```

Comment:
I would follow the documentation style of the rest of this file; three `///` (for doc comments) and a description of the specific command.